demisto_sdk/commands/download/tests/tests_env/content/Packs/TestPack/PreProcessRules/preprocessrule-dummy.yml

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
* can you please switch to spaces instead of tabs? the red marks in Github are disturbing
* why does the extension say `yml` and not `json`?

Reply:
https://github.com/demisto/demisto-sdk/commit/7ed624c4a8fc6f83b1932c50fe4155c71844dc45